site/en/configure/coverage.md

```diff
@@ -188,8 +188,60 @@ py_test(
     ],
 )
 ```
-<!-- TODO: Allow specifying a target for `PYTHON_COVERAGE`, instead of having to use `$(location)` -->
 
+If you are using a hermetic python toolchain, instead of adding an attribute to
+every `py_test` rule you can instead add the coverage tool to the toolchain
+configuration.
+
+Because the `pip_install` rule depends on the python toolchain, you cannot use
+it to fetch the `coverage` module.  Instead, add in your `WORKSPACE` e.g.
+
+```starlark
+http_archive(
+    name = "coverage_linux_x86_64"",
+    build_file_content = """
+filegroup(
```

Comment:
Rather than everyone copy/pasting this same filegroup target, it'd be better if there was a simple, shared, function people could use. That it's a filegroup, and the details of what's in the file group, are implementation details of the coverage library itself; users shouldn't have to worry about that.

Reply:
Agreed, in general.  However those details will also depend on the details of how one is obtaining the coverage library.  The example I put here is what I'd expect would be one of the more common ways people might obtain it, but it is _not_ how our internal teams would do it.  Some teams might for example vendor it into their workspace, others might get it from a local repository rooted in `/usr/lib/python3.x/site-packages` or whatever.